pkg/ui/react-app/src/thanos/pages/blocks/Blocks.tsx

```diff
@@ -21,10 +21,27 @@ export interface BlockListProps {
 
 export const BlocksContent: FC<{ data: BlockListProps }> = ({ data }) => {
   const [selectedBlock, selectBlock] = useState<Block>();
-
-  const { blocks, label, err } = data;
+  const linkRef: any = useRef();
```

Comment:
```suggestion
  const linkRef = useRef<HTMLSpanElement>();
```

Let's try using the proper types if possible. :)